scripts/data/create_task_table.py

```diff
@@ -137,14 +137,9 @@ def get_ds_stats(hf_hub_name):
 # Select all tasks
 for task in MTEB().tasks:
     print("Task: ", task)
-    if "dataset" in task.metadata_dict:
-        hub_name = hub_url = task.metadata.dataset["path"]
-        ds_stats = get_ds_stats(hub_name.split("/")[-1])
-    elif "beir_name" in task.metadata_dict:
-        hub_name = hub_url = "BeIR/" + task.metadata_dict.get("beir_name")
-        ds_stats = get_ds_stats_beir("/".join(hub_name.split("/")[1:]))
-        if "cqadupstack" in hub_name:
-            hub_url = "BeIR/cqadupstack-qrels"
```

Comment:
Nice. Just did a search on `beir_name` and nothing returned. Might have missed removing this part previously.